pyproject.toml

```diff
@@ -162,7 +162,7 @@ extra-standard-library = [
     "pyexpat",
     "zoneinfo",
 ]
-known-first-party = ["_metadata", "_utils"]
+known-first-party = ["ts_utils", "_utils"]
```

Comment:
Can we remove `_utils`?

Reply:
tbf I only named it `_utils` to reflect this config, and as it is right know, it wouldn't even affect the order as those scripts are only using stdlib.

I also see no issue moving or renaming `scripts/sync_protobuf/_utils.py` if that's wanted.